### ISSUE TITLE
[BUG] Getting Slack posts about changed gyms

### ISSUE THREAD

**Issue by majuss** [kvangent/PokeAlarm#181]
## Problem Description
Getting Slack Posts about gyms even if they are all disabled in alarms.json. Interesting thing is, they are getting posted into the #general channel.

## Your Environment
Operating System:  Debian stable
Python Version: 2.7.9
PokeAlarm startup command: runwebhook.py

Did the issue persist after restarting the script?: yes

## Your Alarms Config
```
 {
                        "active": "True",
                        "type":"slack",
                        "api_key":"key",
                        "map": {
                           "enabled":"true",
                           "width":"300",
                           "height":"150",
                           "maptype":"roadmap",
                           "zoom": "15"
                        },
                        "pokemon":{
                          "channel":"pokamoon",
                          "username":"<pkmn>",
                          "icon_url" : "https://raw.githubusercontent.com/kvangent/PokeAlarm/master/icons/<id>.png",
                          "title":"<pkmn> spawned",
                          "url":"<gmaps>",
                          "body": "Verfuegbar: <24h_time> (<time_left>)."
                        }
                },

  "gyms":{
                "To_Valor":"False",
                "To_Mystic":"False",
                "To_Instinct":"False",
                "From_Valor":"False",
                "From_Mystic":"False",
                "From_Instinct":"False"
        },

```
## Troubleshooting Checklist
[x] Most recent version of PokemonGo-Map
[x] Most recent version of PokeAlarm
[x] Checked the Wiki page for service in question

**Comment by shadi1989** [comment]
Got the same issue with Telegram/Twitter while they are FALSE in alarms.json

**Comment by kvangent** [comment]
@majuss @shadi1989 Should be fixed as of commit 2fdb2fcc6b07339a096b9c24374d8998cddf05be
